### DISCUSSION TITLE
Migrate markdown editor.

### DISCUSSION THREAD

**Discussion by lukebp** [decred/politeiagui#2574] [General]
This discussion is replacing #2493.

The markdown editor that politeiagui currently uses, react-mde, is no longer maintained and the repo is now archived. politeiagui will need to migrate to a new markdown editor that is actively maintained.

**Comment by victorgcramos** [comment]
I'd suggest this one here: https://github.com/uiwjs/react-markdown-editor. Looks good, well maintained, and very flexible.

**Comment by victorgcramos** [comment]
New markdown editor added by: https://github.com/decred/politeiagui/pull/2741
